app/kumactl/cmd/config/config_control_planes_add.go

```diff
@@ -35,7 +36,7 @@ func newConfigControlPlanesAddCmd(pctx *kumactl_cmd.RootContext) *cobra.Command
 			if err := config.ValidateCpCoordinates(cp); err != nil {
 				return err
 			}
-			if !cfg.AddControlPlane(cp) {
+			if !cfg.AddControlPlane(cp, args.overwrite) {
```

Comment:
`cfg.AddContext(ctx)` below needs similar treatment.

The difference between `ControlPlane` and `Context` is the following:
* `ControlPlane` points to a Control Plane
* `Context` associates extra info with a given Control Plane, e.g. user credentials, default mesh, etc

At the moment, we don't really use `Context` (to keep it simple from user perspective). But eventually we will.

Reply:
Done